internal/website/README.md

```diff
@@ -0,0 +1,9 @@
+Source for the TODO website. Powered by [Hugo](https://gohugo.io/).
```

Comment:
Why TODO? Don't we know it's gocloud.dev?

Reply:
Most likely, yes. I figured we can fill this in later when we're ready to push it out.